index.bs

```diff
@@ -2383,20 +2392,22 @@ loaded</dfn> steps given |context| and |navigation status|:
 The [=remote end event trigger=] is the <dfn export>WebDriver BiDi load
 complete</dfn> steps given |context| and |navigation status|:
 
- 1. If the [=current session=] is null, return.
+1. [=Queue a webdriver task=] to run the following steps:
 
- 1. Let |params| be the result of [=get the navigation info=] given |context|
-    and |navigation status|.
+  1. If the [=current session=] is null, return.
 
- 1. Let |body| be a [=map=] matching the <code>BrowsingContextLoadEvent</code>
-    production, with the <code>params</code> field set to |params|.
+  1. Let |params| be the result of [=get the navigation info=] given |context|
+     and |navigation status|.
 
- 1. Let |related browsing contexts| be a set containing |context|.
+  1. Let |body| be a [=map=] matching the <code>BrowsingContextLoadEvent</code>
+     production, with the <code>params</code> field set to |params|.
 
  1. Let |navigation id| be |navigation status|'s id.
```

Comment:
Ooops, we now have the same step twice. Some copy-pasting going wrong while reindenting?